state/state.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/roboll/helmfile/environment"
 	"github.com/roboll/helmfile/event"
 	"github.com/roboll/helmfile/tmpl"
+	"github.com/tatsushid/go-prettytable"
```

Comment:
There's a small chance I'd change this to use another table printer, but I do believe printing tables for better summarization is a great idea! Thanks about that.

Reply:
Yes, I think that is an old library that was never released so I guess we can use another one.
this one `github.com/olekukonko/tablewriter v0.0.1` is pretty similar and has a fixed release.